README.md

```diff
@@ -64,80 +64,64 @@ Make sure you have your LambdaTest credentials with you to run test automation s
 - For **Linux/macOS**:
 
   ```bash
+  export PROJECT_TOKEN="xxxxx#java-sdk-testing"
   export LT_USERNAME="YOUR_USERNAME"
   export LT_ACCESS_KEY="YOUR ACCESS KEY"
   ```
 
   - For **Windows**:
 
   ```bash
+  set PROJECT_TOKEN="xxxxx#java-sdk-testing"
   set LT_USERNAME="YOUR_USERNAME"
   set LT_ACCESS_KEY="YOUR ACCESS KEY"
   ```
 
-## Run Your First Test
+### Execute SmartUI Java Tests using smartui-cli
```

Comment:
Please change smartui-cli to SmartUI SDK here

Reply:
but we are using cli to execute the test

Reply:
Okay